round4/team-06.md

```diff
@@ -0,0 +1,140 @@
+
+## Project Management
+
+### Collaboration and Task Allocation
+
+Leader: Emma Lee
+Manager: Rohan Kamat
+Remaining Team Members: Annie Wang, Kevin Lee
+
+- [Kevin] Repository Setup and establishing Chrome extension foundation
+  - Kevin will spearhead the initial setup of our project repository, which includes configuring the foundation, setting up the basic structure and backbone. 
+- [Emma] User Interface Design
+  - Emma will take charge of designing the user interface, starting with creating the Figma mocks for our extension. Then, she will develop the sidebar features, UI components for the user interface. 
+- [Annie] Web Page Text Extraction 
+  - Annie will be responsible for researching and implementing ways to extract text from webpages. This includes researching the appropriate JavaScript libraries for text extraction, manipulation DOM to access specific text elements, and ultimately implementing text extraction. 
+- [Rohan] OpenAI API Integration and Prompt Engineering 
+  - Rohan will be responsible to initiate the integration with OpenAI API and craft the prompts that will work the best for generating summaries. This includes setting up the API calls within our extension, experimenting with different prompt structures to achieve the best summaries, and ensuring our extension interacts efficiently with the API. 
+
+To ensure effective collaboration and communication throughout the project, we plan on using the following platforms: 
+
+- Github: We will use it for code sharing and allowing everyone to collaborate effectively. 
+- Slack: We will utilize Slack for meeting coordinations, general communications, check-ins, updates, and team discussions. 
+- Google Meets: In addition to in-person meetings, we will be utilizing Google Meets to meet virtually as needed throughout the course of the project. 
+
+### Risks and Mitigation
+
+- Limited time to meet which may cause duplication of work. To prevent this, we will try to meet up during class time to ensure that communication is swift.
+- Having issues with setting up the environment. Because everyone is new with the development of Chrome extensions, there may be some technical issues trying to use ReactJS to create extensions. If there are issues, we can ask the TAs or browse the internet to find solutions or tutorials to set up the environment.
+
+### Milestones and Timeline
+
+<b>Week 1:</b>
+- [Kevin] Task 1: Set up repo & Configure chrome extension foundation (ex: manifest file)
+- [Annie] Task 2: Investigate how to approach grabbing webpage text
+  1. JS library
+  2. Manipulate DOM elements
+  - Decide what specific elements we want to target
+- [Emma] Task 3: Finish Figma mocks & Build sidebar base UI component
+- [Rohan] Task 4: Initiate OpenAI API usage & Start prompt engineering
+
+<b>Week 2:</b>
+- [Kevin, Emma] Task 5: Build additional UI components (buttons, dropdowns, etc.)
```

Comment:
I think the UI component should be done as early as possible, and your team should focus more on the web scrapping / integrating AI part. It also seems that Kevin and Emma's tasks are a bit overlapping, you can use the CSS function in Figma to integrate the CSS easily and it should be done very easily.